app/templates/admin/analytics.jade

```diff
@@ -36,7 +36,7 @@ block content
                 - homeBigMAU += count;
             div.count= homeBigMAU
 
-    ul.nav.nav-tabs
+    ul.nav.nav-tabs(style="border:none")
```

Comment:
As suggested by Matt, accidentally got wrapped into a commit. (Fixes the stray line above our analytics graph on the admin/analytics page 😄 )